### PR TITLE
implemented csv file parsing

### PR DESCRIPTION
**Pull request by AlexeyBavika**

### CODE REVIEW

File: src/main/java/com/example/amazonreviewsapp/mapper/UserMapper.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+
+    public UserDto getUserDtoFromLine(String[] line) {
+        UserDto userDto = new UserDto();
+        userDto.setProductId(line[1]);
```
Comment (boroda4436) [inline src/main/java/com/example/amazonreviewsapp/mapper/UserMapper.java:14]:
```suggestion
        userDto.setProductId(line[PRODUCT_ID_POSITION]);
```
старайся уникати magic numbers

File: src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.example.amazonreviewsapp.model.dto.UserDto;
+import com.example.amazonreviewsapp.service.implementation.CsvServiceImpl;
+import com.example.amazonreviewsapp.service.implementation.FileServiceImpl;
+import static org.junit.Assert.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+
+public class CsvServiceImplTest {
+    private static final String TEST_FILE_PATH = "src\\test\\resources\\testCsv";
```
Comment (dmytro-hryhoruk) [inline src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java:13]: я зможу це запустити якщо в мене не віндовс?

File: src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    private static List<String> testFileContentList;
+
+    @BeforeAll
+    static void initializeList() {
+        FileService fileService = new FileServiceImpl();
+        testFileContentList = fileService.readFile(TEST_FILE_PATH);
+    }
+
+    @Test
+    public void shouldContainTenRecords() {
```
Comment (dmytro-hryhoruk) [inline src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java:24]: let's use unit test naming convention

File: src/test/java/com/example/amazonreviewsapp/service/FileServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package com.example.amazonreviewsapp.service;
+
+import com.example.amazonreviewsapp.service.implementation.FileServiceImpl;
+import static org.junit.Assert.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+
+public class FileServiceImplTest {
+    private static final String TEST_FILE_PATH = "src/test/resources/testCsv";
```
Comment (dmytro-hryhoruk) [inline src/test/java/com/example/amazonreviewsapp/service/FileServiceImplTest.java:10]: you can get file from resources without hardcoding the full path, but only with name https://mkyong.com/java/java-read-a-file-from-resources-folder/

Comment (dmytro-hryhoruk) [reply src/test/java/com/example/amazonreviewsapp/service/FileServiceImplTest.java:10]: small changes in fileService should work

File: src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java [hunk 5]
```diff
@@ -5,23 +5,18 @@
...
-
-    @BeforeAll
-    static void initializeList() {
-        FileService fileService = new FileServiceImpl();
-        testFileContentList = fileService.readFile(TEST_FILE_PATH);
-    }
 
     @Test
     public void Should_Pass_IfListContainsExpectedNumberOfRecordsAfterFileParsing() {
+        File file = new File(getClass().getClassLoader().getResource("testCsv").getFile());
```
Comment (dmytro-hryhoruk) [inline src/test/java/com/example/amazonreviewsapp/service/CsvServiceImplTest.java:17]: do you need to pass File into File constructor?

File: src/test/java/com/example/amazonreviewsapp/service/FileServiceImplTest.java [hunk 6]
```diff
@@ -18,7 +18,8 @@ public static void initialize() {
 
     @Test
     public void Should_Pass_IfListContainsExpectedNumberOfRowsAfterReadingFromFile() {
-        List<String> list = fileService.readFile(TEST_FILE_PATH);
+        File file = new File(getClass().getClassLoader().getResource("testCsv").getFile());
```
Comment (dmytro-hryhoruk) [inline src/test/java/com/example/amazonreviewsapp/service/FileServiceImplTest.java:21]: same

**Pull request merged**